editor/script_editor_debugger.cpp

```diff
@@ -2185,6 +2189,13 @@ void ScriptEditorDebugger::_item_menu_id_pressed(int p_option) {
 	}
 }
 
+void ScriptEditorDebugger::_tab_changed(int p_tab) {
+	if (p_tab == 5) {
```

Comment:
This uses a magic number, what would be a good way to solve this? Would a `const int` or `#define` be considered sufficient?

Reply:
I guess you could use `tabs->get_table_title(p_tab) == TTR("Video RAM")`. That's still arbitrary and could break if it gets renamed, but it's a bit safer than a magic number.

Reply:
i think both solutions are terrible, it adds a lot of unpredictability in the future, if someone ever makes any changes here, and if someone ever breaks this by accident, they won't realize what's causing problems, it's not intuitive at all

i would avoid a fixer number, or a fixed string - if you really have to, at least copy the current string, instead of manually typing it by hand